README.md

```diff
@@ -58,3 +58,4 @@ Where you replace `83b1380` with the sha you want to use.
 
 [build-status-image]: https://github.com/getsentry/self-hosted/workflows/test/badge.svg
 [build-status-url]: https://git.io/JUYkh
+
```

Comment:
🤣 Why not literally add a disclaimer in the readme: Sentry is not vulnerable to log4shell

Reply:
Because nobody is going to care in a month? 0, 1, infinity.